salsa_dancing_molecules/simulations/simulation.py

```diff
@@ -1,4 +1,68 @@
 """Module for running a simulation."""
+from ..lennardjonesparse import parse_lj_params
+from ase.md.velocitydistribution import MaxwellBoltzmannDistribution
+from ase.md.verlet import VelocityVerlet
+from ase.md.andersen import Andersen
+from ase import units
+from asap3 import Trajectory
+from ..variables import Variables
+
+
+def choose_potential(potential, kim_model, use_asap, atoms):
+    """Change the atoms object according to what potential is used.
+
+    Args:
+        potential - String with name of potential to be used.
+        kim_model - String with ID for ensemble to be chosen from openkim.
+        use_asap - boolean if asap3 is to be used instead of ASE
+        atoms - atoms object to be edited.
+    """
+    potential = potential.lower().strip()
+    if potential == "openkim":  # use openkim
+        try:
+            from ase.calculators.kim import KIM
+            atoms.calc = KIM(kim_model)
+        except RuntimeError:
+            print(f'{kim_model} is not a valid OpenKIM potential, '
+                  'will use standard Lennard-Jones potential')
+            potential = 'lennard-jones'
+    if potential == "lennard-jones":
+        element_symbols = atoms.get_chemical_symbols()
+        if len(element_symbols) > 1:
+            print('More than one element was inputted, will use '
+                  f'LJ-parameters for {element_symbols[0]}')
+        element, rc, epsilon, sigma = parse_lj_params(element_symbols[0])
+
+        if use_asap:
+            from asap3 import LennardJones
+            atoms.calc = LennardJones(element, epsilon, sigma, rCut=rc,
+                                      modified=True)
+        else:
+            from ase.calculators.lj import LennardJones
+            atoms.calc = LennardJones(epsilon=epsilon, sigma=sigma, rc=rc)
+
+
+def choose_ensemble(ensemble, target_temperature, atoms):
+    """Create dyn object depending on what ensemble to be used.
```

Comment:
Could you write `Create an ASE dynamics object...` instead?  It is not clear to me what "dyn object" means without reading the code.